app.js

```diff
@@ -43,17 +59,33 @@ let bullet = {
   rotation: 0,
 };
 
-let enemy2X = 1600;
-let enemy3X = 2600;
-let enemy2Y = 81;
-let enemy3Y = 281;
-let rocketX = 20;
-let rocketY = 70;
+let isShipMovingUp = false;
+let isShipMovingLeft = false;
+let isShipMovingRight = false;
+let isShipMovingDown = false;
+let enemyHealth50 = {
+  element: document.querySelector('#healthBar50')};
```

Comment:
Please put the closing curly brace `}` on the new line.
Like this:
```
let enemyHealth50 = {
  element: document.querySelector('#healthBar50')
};
```

Please do this for all variables you added.